source/_sass/infeedl.scss

```diff
@@ -1,18 +1,19 @@
 @font-face {
-  font-family: "gotham_proregular";
-  src: url("../../assets/fonts/gothaproreg-webfont.eot");
-  src: url("../../assets/fonts/gothaproreg-webfont.eot?#iefix") format("embedded-opentype"), url("../../assets/fonts/gothaproreg-webfont.woff") format("woff"), url("../../assets/fonts/gothaproreg-webfont.ttf") format("truetype");
-  font-weight: normal;
+  font-family: "AvenirNext Demibold";
+  src: url('../../assets/fonts/AvenirNextCyr-Demi..eot?#iefix') format('embedded-opentype'),  url('../../assets/fonts/AvenirNextCyr-Demi.otf')  format('opentype'),
```

Comment:
Commas in lists should be followed by a single space<br>Prefer double-quoted strings